src/main/java/seedu/souschef/logic/parser/contextparser/RecipeParser.java

```diff
@@ -83,6 +84,9 @@
         case SelectCommand.COMMAND_WORD:
             return new SelectCommandParser().parseIndex(recipeModel, arguments);
 
+        case SurpriseCommand.COMMAND_WORD:
```

Comment:
Make use of ur feature parser. By changing the call from getCrossContextCommand() in appContentParser.